src/screens/Landing/index.js

```diff
@@ -2,15 +2,17 @@ import React from 'react';
 import InfoSection from './InfoSection';
 import Subscribe from './Subscribe';
 import Footer from '../../components/common/Footer';
+import Navbar from '../../components/common/Navbar/Navbar';
 
 const Dashboard = () => {
-	return (
-		<>
-			<InfoSection />
-			<Subscribe />
-			<Footer />
-		</>
-	);
-};
+  return (
+    <>
```

Comment:
It's a good practice to wrap it into a React Fragment, Anyway I am merging it,

Reply:
Okay is that something I still need to fix or did you change it? thanks